bgpd/bgp_vty.c

```diff
@@ -114,6 +114,7 @@ FRR_CFG_DEFAULT_ULONG(BGP_KEEPALIVE,
 	{ .val_ulong = 3, .match_profile = "datacenter", },
 	{ .val_ulong = 60 },
 )
+FRR_CFG_DEFAULT_ULONG(BGP_DELAYOPEN, {.val_ulong = 120}, )
```

Comment:
Shouldn't this be set lower or unset for the datacenter profile?

Reply:
it is not activated by default, so users will most likely set an other value if they use it anyways.

Reply:
I don't understand still why do we need to add FRR_CFG_DEFAULT_ULONG if this is the same for all versions and profiles. Can't we reuse BGP_DEFAULT_DELAYOPEN?